content/applications/hr/fleet/models.rst

```diff
@@ -0,0 +1,195 @@
+- :guilabel:`CO2 Emissions`: enter the average carbon dioxide emissions the vehicle produces in
+  grams per kilometer (g/km). This information is provided by the car manufacturer.
+- :guilabel:`CO2 Standard`: enter the standard amount of carbon dioxide in grams per kilometer
+  (g/km) for a similar sized vehicle.
+- :guilabel:`Transmission`: select :guilabel:`Manual` or :guilabel:`Automatic` transmission type
+  from the drop-down menu.
+- :guilabel:`Power`: if the vehicle is electric or hybrid, enter the power the vehicle uses in
+  kilowatts (kW).
+- :guilabel:`Horsepower`: enter the vehicle's horsepower in this field.
+- :guilabel:`Horsepower Taxation`: enter the amount that is taxed based on the size of the vehicle's
+  engine. This is determined by local taxes and regulations, and varies depending on the location.
+  It is recommended to check with the accounting department to ensure this value is correct.
+- :guilabel:`Tax Deduction`: this field auto-populates according to the engine specifications, and
+  cannot be modified. The percentage is based on the localization settings and local tax laws.
+
+Vendors tab
+-----------
+
+Specify the vendors a vehicle can be purchased from in this tab. With proper setup, requests for
+quotations for vehicles can be easily created through Odoo's *Purchase* app.
+
+The vendors the vehicle can be purchased from may be added in this tab. To add a vendor, click
+:guilabel:`Add`, which opens a pop-up window with a list of all the vendors currently in the
+database. Select the vendor to add by clicking the checkbox next to the vendor name, then click
+:guilabel:`Select`. There is no limit to the number of vendors that can be added to this list.
+
+If a vendor is not in the database, add a vendor by clicking :guilabel:`New`. In a new vendor form,
+enter the information in the vendor tab, then click :guilabel:`Save & Close` to add the vendor and
+close the window, or :guilabel:`Save & New` to add the current vendor and create another new vendor.
+
+.. image:: models/vendor.png
+   :align: center
+   :alt: Vendor form to fill out when adding a new vendor.
+
+.. _fleet/categories:
+
+Model Category
+==============
+
+To best organize a fleet, it is recommended to have vehicle models housed under a specific category,
+to more easily see what kinds of vehicles are in the fleet. Model categories are set on the
+:ref:`vehicle model form <fleet/add-model>`.
+
+Odoo does **not** come with any models preconfigured; all models must be added.
+
+To view any models currently set up in the database, navigate to :menuselection:`Fleet app -->
+Configuration --> Models: Categories`. All models are displayed in a list view.
+
+Add a new model category
+------------------------
+
+To add a new category, click the :guilabel:`New` button in the top-left corner. A new entry line
+appears at the bottom of the list. Type in the new category, then click :guilabel:`Save` to save the
+entry.
```

Comment:
I totally see the 'Save' button appear, but when I clicked away the new entry stayed, as well. So, that might be worth mentioning...